Assets/Scripts/Controllers/MouseController.cs

```diff
@@ -369,6 +396,26 @@ void UpdateDragging()
                             {
                                 isValid = true;
                             } 
+                        } else if (dragType == "path")
```

Comment:
Maybe move them to a dictionary of functions so that Lua mods can add new amazing ways of laying down furniture?